conf/nginx-alldomains.conf

```diff
@@ -26,9 +26,22 @@
 		index index.php;
 		alias /usr/local/lib/roundcubemail/;
 	}
-	location ~ /mail/config/.* {
-		# A ~-style location is needed to give this precedence over the next block.
-		return 403;
+	location ~ ^/mail/(config|temp|logs)/ {
+        	deny all;
+	}
+	location ~ ^/mail/(README|INSTALL|LICENSE|CHANGELOG|UPGRADING)$ {
+        	deny all;
+	}
+	location ~ ^/mail/(bin|SQL)/ {
+        	deny all;
+	}
+	location ~ ^/mail/(.+\.md)$ {
+        	deny all;
+	}
+	location ~ ^/mail/\. {
+        	deny all;
+        	access_log off;
```

Comment:
I can't find a reference to disabling the access logs. (I'm not fluent in htaccess) shouldn't we just log the deny's until they become a problem? Perhaps they can be used for fail2ban.